content/docs/command-reference/import-db.md

```diff
@@ -0,0 +1,153 @@
+| **MySQL**           | `mysqlclient`                     | `mysql://{username}:{password}@{hostname}/{database_name}`                                          |
+| **Oracle**          | `cx_Oracle`                       | `oracle://{username}:{password}@{hostname}/{database_name}`                                         |
+| **PostgreSQL**      | `psycopg2`                        | `postgresql://{username}:{password}@{hostname}/{database_name}`                                     |
+| **Snowflake**       | `snowflake-sqlalchemy`            | `snowflake://{user}:{password}@{account}.{region}/{database}?role={role}&amp;warehouse={warehouse}` |
+| **SQLite**          | -                                 | `sqlite://path/to/file.db`                                                                          |
+| **SQL Server**      | `pyodbc`                          | `mssql+pyodbc://{username}:{password}@{hostname}:{port}/{database_name}`                            |
+| **Trino**           | `trino`                           | `trino://{username}:{password}@{hostname}:{port}/{catalog}`                                         |
+
+DVC uses `sqlalchemy` internally. So DVC should support any SQL databases that
+provide dialects for SQLAlchemy. Refer to their documentation for more details.
+
+## Database Permissions
+
+It is recommended to use an user account with limited access to databases and
+tables and one that only has minimal read-only privileges like running `SELECT`
+queries. Different databases have different approaches to this. Refer to their
+documentation for more details.
+
+## Options
+
+- `-o <path>`, `--out <path>` - specify a `path` to the desired location in the
+  workspace to place the file. The default filename will be `results`, with an
+  extension name added based on `--output-format`.
+
+- `--sql <query>` - SQL query to run. You can also use a connection string set
+  in config, with `--conn`.
+
+- `--output-format` - type of format to materialize into. `csv` (default) and
+  `json` is supported.
+
+- `--conn connection` - Name of the database connection to use. The connection
+  has to be set in the
+  [config](/doc/user-guide/project-structure/configuration#db).
+
+- `-f`, `--force` - when using `--out` to specify a local target file or
+  directory, the operation will fail if those paths already exist. this flag
+  will force the operation causing local files/dirs to be overwritten by the
+  command.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+### Downloading sql query result
+
+To import from a database using a `db` config set:
+
+```dvc
+$ dvc import-db --sql "select * from customers" --conn pgsql
```

Comment:
Do we need to do something to the syntax highlighter to support new commands @yathomasi?

Reply:
Yes, we need to add it to https://github.com/iterative/gatsby-theme-iterative/blob/main/packages/gatsby-theme-iterative/config/prismjs/dvc-commands.js